core/package.json

```diff
@@ -23,6 +23,285 @@
   "collection:main": "dist/collection/index.js",
   "collection": "dist/collection/collection-manifest.json",
   "types": "dist/types/interface.d.ts",
+  "exports": {
+    ".": {
```

Comment:
We'll need to additionally export the `./dist/docs.json`.

e.g.:
```json
"./docs.json": {
      "import": "./dist/docs.json",
      "require": "./dist/docs.json"
    },
```

There are 3 scripts in the vue project that reference the `dist/docs.json` path that will need to be updated to remove `dist` from the path.

There is an error you will see locally with Vue that you can ignore for now. I am working with the team to make them aware (exists in `main`):
```
(!) Plugin typescript: @rollup/plugin-typescript TS2315: Type 'Plugin_2' is not generic.
src/ionic-vue.ts: (24:24)

24 export const IonicVue: Plugin<[IonicConfig?]> = {
```